multicluster/apis/multicluster/v1alpha1/webhook_suite_test.go

```diff
@@ -82,18 +83,6 @@ var _ = BeforeSuite(func() {
 	err = admissionv1beta1.AddToScheme(scheme)
 	Expect(err).NotTo(HaveOccurred())
 
-	err = admissionv1beta1.AddToScheme(scheme)
```

Comment:
Why these are no longer needed?

Reply:
they are duplicate codes, I think it's a kubebuilder bug. it will add duplicate codes when we add some new resource into one API group.

Reply:
Ok.